### ISSUE TITLE
Running on 10X multiome?

### ISSUE THREAD

**Issue by ktpolanski** [cortes-ciriano-lab/SComatic#6]
Hello,

We've got a bunch of data that we'd like to run SComatic on. Some of the samples are 10X multiome, i.e. both GEX and ATAC for the same samples. The data was processed via cellranger-arc, the standard practice for 10X multiome, yielding separate BAMs for GEX and ATAC.

Do you have any suggestions for how to proceed? The BAM files for the two are heterogeneous. The GEX has `CB`, `NH` and `nM` tags, while the ATAC has `CB` and an `NM` (not sure if this is the same as `nM` from the GEX). Would it make sense to merge the two BAMs together given this disparity?

**Comment by Francesc-Muyas** [comment]
Dear user, 
I strongly suggest not merging both bam files, as one is RNA-based approach, and the other is DNA-based. There are differences in the way of processing the bam files, as well as biases that depend on the RNA/DNA approach. 

Regarding how to process the bam files, I would follow the [toy example](https://github.com/cortes-ciriano-lab/SComatic/blob/main/docs/SComaticExample.md) as a template for the scRNA-seq data. 

For ATAC, I would follow a similar approach but with minor changes:
- Set the __--min_MQ_ 30_ parameter in the _SplitBamCellTypes.py_ script
- Do not use the RNA-editing sites in the _Step 4.2_ 
- Use the [scATAC-seq PoN](https://github.com/cortes-ciriano-lab/SComatic/tree/main/PoNs) in the _Step 4.2_

I hope it helps, 
Fran

**Comment by ktpolanski** [comment]
Thank you for your response, I've been mucking around with the GEX portion to get a feel for the processing. Step two's parallelisation is very nice, is there some way to apply similar principles to step 1 and 4.1?

Given the absence of `nM` and `NH` tags from the ATAC, I presume I just ignore those in step 1. Or would the ATAC's `NM` work as an adequate replacement for `nM`?

**Comment by Francesc-Muyas** [comment]
Dear user, 
Thanks for your suggestion. We have been thinking about the implementation of extra parallelisation in other steps of the tool, but this is something that will be addressed in the future. 

Regarding your second question: in the current version of SComatic, if you do not have the exact _nM_ or _NH_ tags in the bam files, you should not use these filters. However, we will work to implement the possibility of using _NM_ or _nM_ depending on the input bam file. We will put this suggestion in our TODO list. 

Thanks for your feedback, 
Fran

**Comment by ktpolanski** [comment]
I've had a chat with @apredeus, who has more genomic experience than me. He opined that the MQ filter should be sufficient for the ATAC, and I proceeded as such. The coworker that got the ATAC results was happy with them.

Here's a master list of the tweaks that were made relative to the GEX demo, following Fran's suggestions a few comments back:
- Step 1: `--max_nM 5` and `--max_NH 1` removed, `--min_MQ 30` added
- Step 2: `--min_mq 30` added
- Step 4.2: `--editing` removed, and `--pon` altered to point at the ATAC one

Given the fact we're interested in finding mutations shared across some cell populations, I also drastically increased `--max_cell_types` in both GEX and ATAC processing, but that's specific to our question.

Thanks a lot for this tool! Seeing how we've got multiple 10X samples for each donor, I was able to speed up step 1 locally by running it in parallel on each sample's BAM separately and then merging per cell type.
